CoreFoundation/Preferences.subproj/CFPreferences.c

```diff
@@ -51,7 +51,7 @@ CONST_STRING_DECL(kCFPreferencesCurrentUser, "kCFPreferencesCurrentUser")
 
 
 static CFAllocatorRef _preferencesAllocator = NULL;
-CF_PRIVATE CFAllocatorRef __CFPreferencesAllocator(void) {
+CF_EXPORT CFAllocatorRef __CFPreferencesAllocator(void) {
```

Comment:
Do we use these from the Foundation binary? Otherwise they seem like they should be internal.

Reply:
Precisely :-)  They are SPI for Foundation.